forecast-admin/forecast/opportunities/management/commands/load_opportunities.py

```diff
@@ -0,0 +1,193 @@
+import os
+import csv
+from datetime import date
+from decimal import Decimal, InvalidOperation
+import re
+
+from django.core.management.base import BaseCommand, CommandError
+from django.core.exceptions import ValidationError
+from django.db import transaction
+from opportunities.models import Office, Opportunity, OSBUAdvisor
+from optparse import make_option
+
+
+# A data loader of contract opportunities
+# Thank you Brendan Sudol... :)
+# https://github.com/brendansudol/calc/blob/ca4a13d0fa34a468d788710975bf71959eb8bb20/contracts/management/commands/load_s70.py
+class Command(BaseCommand):
+    help = 'Load data from CSV'
+
+    default_filename = 'forecast/fixtures/fy16q1.csv'
```

Comment:
The CSV file for the data, shouldn't be in the fixtures directory. Fixtures are a specific thing in Django. If anything, put it in a data/ directory. 

Even better: put the csv file in a data directory, but also change this management command to accept the file name on the command line.